charts/loki-canary/Chart.yaml

```
@@ -3,7 +3,7 @@ name: loki-canary
 description: Helm chart for Grafana Loki Canary
```

@JStickler 

Thanks for the contribution. The helm-doc render would README.md.gotmpl -> read this line (`.chart.description`) -> output to README.md, so:

```suggestion
description: Helm chart for Grafana Loki Canary. This is a community maintained chart."
```